wazuh/wazuh-dashboard-plugins
# Add JSON visualization to Vulnerability Detection inventory flyout

### Description
This PR adds a tab with the JSON view of a document inside the Vulnerability detection inventory flyout.
 
### Issues Resolved
#6589 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/42900763/5290f9e7-7d20-4e01-a3c9-1a7064d7318d)
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/42900763/2374eb8b-212e-40ef-a656-5b4c4dd9616b)


### Test

- Run the 'scripts/vulnerabilities-events-injector/dataInjectScript.py' to inject vulnerabilities to the environment
- Navigate to Vulnerability Detection section, and Inventory tab
- Click on the Inspect details icon of a row, to open the Flyout
- Verify that the flyout contains a `Table` and a `JSON` sections. Verify that the information shown by both tabs is the same, and corresponds to the selected row.

**If no vulnerabilities are shown it may be because the manager has the cluster disabled. Follow this steps to enable it**
- Attach a console to the manager
- Run the following commands
```
sed -i 's/<node>NODE_IP<\/node>/<node>127.0.0.1<\/node>/' /var/ossec/etc/ossec.conf
sed -i 's/<key><\/key>/<key>c98b62a9b6169ac5f67dae55ae4a9088<\/key>/' /var/ossec/etc/ossec.conf
sed -i 's/<disabled>yes<\/disabled>/<disabled>no<\/disabled>/' /var/ossec/etc/ossec.conf 
/var/ossec/bin/wazuh-control restart
```

### Check List
